src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/components/BootableVolumeRow/BootableVolumeRow.tsx

```diff
@@ -2,45 +2,67 @@ import React, { FC } from 'react';
 
 import { getTemplateOSIcon as getOSIcon } from '@catalog/templatescatalog/utils/os-icons';
 import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
-import { V1alpha2VirtualMachineClusterPreference } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  V1alpha1PersistentVolumeClaim,
+  V1alpha2VirtualMachineClusterPreference,
+} from '@kubevirt-ui/kubevirt-api/kubevirt';
 import { ANNOTATIONS } from '@kubevirt-utils/resources/template';
+import { NO_DATA_DASH } from '@kubevirt-utils/resources/vm/utils/constants';
 import { Text, TextVariants } from '@patternfly/react-core';
-import { Td, Tr } from '@patternfly/react-table';
+import { Tr } from '@patternfly/react-table';
+
+import { humanizeBinaryBytes } from '../../../../../../../utils/utils/humanize';
```

Comment:
import? guessing the auto import did a mistake

Reply:
from some reason I got minified react error when using `@kubevirt-utils/utils/humanize`
any ideas why?